src/pages/angels.js

```diff
@@ -39,17 +35,10 @@ const AngelsPage = () => {
       <div>
         <div>
           <AngelCard
-            weddingAnniversary="Paper"
-            priceText="100 NOK"
-            onClick={() => placeOrder(SILVER_SKU)}
-          ></AngelCard>
-        </div>
-        <div>
-          <AngelCard
-            weddingAnniversary="Bronze"
-            priceText="800 NOK"
+            weddingAnniversary="Bronze Angel"
+            priceText="500 NOK"
             spotsText="15 spots"
-            description="Lifetime access to POW! for one person."
+            description1="Lifetime access to POW! for one person."
             buttonText="Select"
             skuProp={BRONZE_SKU}
```

Comment:
You do not need `skuProp` on AngelCard, they are not used.

Reply:
Check